sqlmesh/core/context.py

```diff
@@ -829,7 +828,7 @@ def get_dag(self, format: str = "svg") -> graphviz.Digraph:
 
         graph = graphviz.Digraph(node_attr={"shape": "box"}, format=format)
 
-        for name, upstream in self.dag.graph.items():
+        for name, upstream in self.dag._graph.items():
```

Comment:
Why did we make `_graph` private and are now accessing it from the outside?

Reply:
because by default graph does a copy. should i make it not copy by default?